xds/src/main/java/io/grpc/xds/WeightedTargetLoadBalancerProvider.java

```diff
@@ -185,7 +185,7 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-      return Objects.hash(targets);
+      return Objects.hashCode(targets);
```

Comment:
I am confused. In @dapengzhang0 's recent PR, he explicitly changed usages of `Objects.hashCode()` to `Objects.hash()`. So which one on earth should we use?

Reply:
according to the linter both generates same value. but `hash` accepts vararg, so it creates unnecessary objects when calling this method. the object is not really used in the hash based collection so it doesn't really matter tbh.

Reply:
The recent PR was changing from guava Objects to java.util.Objects. It's a different issue.